pkg/kubelet/controller.go

```diff
@@ -155,6 +155,18 @@ func (c *Controller) nodeAddress(node v1.Node) (string, map[v1.NodeAddressType][
 	return "", m, fmt.Errorf("host address unknown")
 }
 
+// nodeIsReady checks the node for the Ready condition to see if it is set to
+// "True". Without this condition status, we cannot guarantee the node is
+// available and ready to be a target.
```

Comment:
This language is too strong; it implies that with this condition status, we can guarantee the node is available and ready.

```suggestion
// nodeIsReady checks the node for the Ready condition to see if it is set to
// "True". Unless the node is ready, it's unlikely that we can scrape it.
```